addons/resource.language.en_gb/resources/strings.po

```diff
@@ -8488,7 +8488,7 @@ msgid "Reset PVR database"
 msgstr ""
 
 msgctxt "#19186"
-msgid "All data in the PVR database is being erased. Are you sure?"
+msgid "All data in the PVR database will be erased. Are you sure?"
 msgstr ""
```

Comment:
I think @MartijnKaijser's change is fine as it is, no need to change everything.